app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,5 @@
     <string name="title_current">Current</string>
     <string name="title_forecast">Forecast</string>
     <string name="title_history">History</string>
+    <string name="description_weather">Weather</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.description_weather` appears to be unused
 Unused resources make applications larger and slow down builds.